x/fundraising/keeper/auction.go

```diff
@@ -490,14 +490,22 @@ func (k Keeper) CalculateBatchAllocation(ctx sdk.Context, auction types.AuctionI
 		MatchedLen:         0,
 		MatchedPrice:       sdk.ZeroDec(),
 		TotalMatchedAmount: sdk.ZeroInt(),
+		AllocationMap:      map[string]sdk.Int{},
+		ReservedMatchedMap: map[string]sdk.Int{},
+		RefundMap:          map[string]sdk.Int{},
 	}
 	allowedBiddersMap := auction.GetAllowedBiddersMap() // map(bidder => maxBidAmt)
 	allocationMap := map[string]sdk.Int{}               // map(bidder => allocatedAmt)
 	reservedMatchedMap := map[string]sdk.Int{}          // map(bidder => reservedMatchedAmt)
 
+	for _, ab := range auction.GetAllowedBidders() {
+		mInfo.AllocationMap[ab.Bidder] = sdk.ZeroInt()
+		mInfo.ReservedMatchedMap[ab.Bidder] = sdk.ZeroInt()
+	}
+
 	// Iterate from the highest matching bid price and stop until it finds
 	// the matching information to store them into MatchingInfo object
-	for _, bid := range bids {
+	for _, bid := range bids { // (1, 500), (0.9, 500), (0.8, 500)
```

Comment:
I will remove the comment later.

Reply:
Yes, when this PR gets R4R I suggest you to remove all the unnecessary comments and unused codes.